2020_Fall/readme.md

```diff
@@ -25,7 +25,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Cerda, Daniel
 
-- [ ] Chandrasekhar, Dhruva
+- [X] Chandrasekhar, Dhruva (@dhruvakc) 🎠🎮🎃
```

Comment:
Hi @dhruvkc - looks like you've got a type in your GitHub username. Please edit this existing Pull request to add another commit.

Reply:
@joewadcan thank you for flagging, fixed the typo and added the commit!